The rename argument renPdb=true causes a null pointer

Confuser 1.4.1 with .NET 4.7.2 Winforms exe
Create a project with only the default rename mode (unicode) and renPdb=true:
`<protection id="rename">
  <argument name="mode" value="unicode" />
  <argument name="renPdb" value="true" />
</protection>`

Run it and get:
`[DEBUG] Renaming...
[ERROR] Unknown error occurred.
Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Confuser.Renamer.RenamePhase.Execute(ConfuserContext context, ProtectionParameters parameters)
   at Confuser.Core.ProtectionPipeline.ExecuteStage(PipelineStage stage, Action`1 func, Func`1 targets, ConfuserContext context)
   at Confuser.Core.ConfuserEngine.RunPipeline(ProtectionPipeline pipeline, ConfuserContext context)
   at Confuser.Core.ConfuserEngine.RunInternal(ConfuserParameters parameters, CancellationToken token)
Failed at 1:27 PM, 0:00 elapsed.`

--- a/Confuser.Renamer/RenamePhase.cs
+++ b/Confuser.Renamer/RenamePhase.cs
@@ -57,7 +57,9 @@ namespace Confuser.Renamer {
 							if (!string.IsNullOrEmpty(local.Name))
 								local.Name = service.ObfuscateName(local.Name, mode);
 						}
-						method.Body.PdbMethod.Scope = new PdbScope();
+
+						if (method.Body.HasPdbMethod)
+							method.Body.PdbMethod.Scope = new PdbScope();
 					}
 				}
 

